docs/WindowsCrossCompile.md

```diff
@@ -0,0 +1,44 @@
+# Cross-compiling Swift for Windows with `clang`.
+
+This document describes how to cross compile Swift on Windows on a non-Windows
+host. For more context on the status of Swift on Windows in general, see
+[the Windows doc](./Windows.md)
+
+## 1. Setup Visual Studio Environment Variables
```

Comment:
The verb phrase is "set up" with a space. Also, sentence case or title case for all the subheadings, but pick one.